v1/contracts/VotingToken.sol

```diff
@@ -0,0 +1,43 @@
+pragma solidity ^0.5.0;
+
+import "./MultiRole.sol";
+
+import "openzeppelin-solidity/contracts/drafts/ERC20Snapshot.sol";
+
+
+/**
+ * @title UMA voting token
+ * @dev Supports snapshotting and allows the Oracle to mint new tokens as rewards.
+ */
+contract VotingToken is ERC20Snapshot, MultiRole {
+
+    enum Roles {
+        // Can set the minter.
+        Governance,
+        // The Oracle contract (currently named Voting.sol) can mint new tokens as voting rewards.
```

Comment:
nit: can we update this comment to be more general now that there can be multiple minters?

Reply:
Done.